components/frontend/src/App.js

```diff
@@ -123,7 +123,7 @@ class App extends Component {
             datamodel: data_model,
             reports: reports.reports || [],
             reports_overview: {
-              layout: reports.layout, subtitle: reports.subtitle, title: reports.title, permissions: reports.permissions },
+              layout: reports.layout, subtitle: reports.subtitle, title: reports.title, permissions: reports.permissions, issueTrackers: reports.issue_trackers || {} },
```

Comment:
I'd prefer consistent casing...

Reply:
I agree.. but then I'm wondering where to insert empty dictionaries if nothing is present. Maybe the best solution is during the database initialization to fill it in all old report_overviews.

Reply:
Agreed.